### PR TITLE
Final changes

### PR DESCRIPTION
**Pull request by Usagib**
final changes for project, updated readme file and added css styling

### CODE REVIEW

File: CSS/style.css [hunk 1]
```diff
@@ -1,13 +1,15 @@
-span{
+/*span{
   font-size: 12px;
   color: red;
```
Comment (samgaco) [inline CSS/style.css]: It's better to delete commented code

**Review by samgaco — CHANGES_REQUESTED** [review]
### Hi @Usagib and @sebastianlujan :vulcan_salute: 

### The project is looking brilliant so far :ok_hand: , some things to address:

* There should be at least three embedded images in the rightmost part of the navbar. In the original page you can find some inspiration:
ok
* **[OPTIONAL]** Good work on the readme file, try to go through a markdown [cheat sheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) to see how to make it look even better.
* Read on semantic HTML elements, you should try to implement it. For example, you could use `<main>` for the dominant content of the body.

### Please, work on these changes and submit back for another code review when you are done. Remember to commit and request a new code review on this branch, no need to open a new one. Regards! :muscle:

**Comment by Usagib** [comment]
Hello Team!, we have made the modifications required! 
please review our code ;)

**Review by Oghenebrume50 — APPROVED** [review]
Hi team, good job, it's is a wrap :smiley:  please go ahead and merge to master. Then submit a project completion form for this project, good luck for the next project. :+1:

**Pull request merged**